R/calcFETaxes.R

```diff
@@ -120,12 +120,16 @@ calcFETaxes <- function(subtype = "taxes") {
   getYears(Rtax) <- "2005"
   getYears(Renergy) <- "2005"
 
+
   # Weights do not take into account the differentiation by services. So if
   # the tax in a Cooling country is very high and the tax in a country in the
   # same region using a lot of electricity for cooking is low, the tax for
   # cooling and cooking with electricity will be equal where it should be high
   # for cooling and low for cooking
   # So, we can assume that countries are app. similar in a given region
 
-  list(x = Rtax, weight = Renergy, unit = "US$2017/GJ", description = desc)
+  list(x = Rtax, weight = Renergy, unit = "US$2017/GJ", description = desc,
+       # do not throw a warning for zero weights, as they seem to be intended
+       aggregationArguments = list(zeroWeight = "allow")
```

Comment:
Allow zero weights to suppress warnings without changing input data. @Renato-Rodrigues LEt me know if this should be dealt with differently (e.g. by using small weights != 0)

Reply:
Talked to Renato and he approved of the current approach.